src/serializer/Emitter.js

```diff
@@ -341,7 +341,9 @@ export class Emitter {
       let argsToRecurse;
       if (val.kind === "conditional") {
         let cf = this._conditionalFeasibility.get(val);
-        invariant(cf !== undefined);
+        if (cf === undefined) {
+          return result;
```

Comment:
I'm worried about relaxing such a generic invariant because of our special case.

Why is this justified?

Reply:
This doesn't seem related to this PR. It's just that we encounter it when a ReactElement is an arg on a generator entry.

Reply:
I don't understand what "doesn't seem related to this PR" means in this case.
This PR changes this line, so to me it seems related. :-)

I'm worried that we're relaxing an existing constraint because of some change that only benefits us. It might mean that we should fix it on our side instead (e.g. by not getting into this code path altogether).

Can you explain what this invariant is checking, and why you think checking it is no longer important in the generic case?